Code/GraphMol/RGroupDecomposition/RGroupCore.h

```diff
@@ -0,0 +1,41 @@
+//
+//  Copyright (C) 2020 Novartis Institutes for BioMedical Research
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#ifndef RGROUP_CORE
+#define RGROUP_CORE
+
+#include "../RDKitBase.h"
+#include "RGroupUtils.h"
+
+namespace RDKit {
+
+//! RCore is the core common to a series of molecules
+struct RCore {
```

Comment:
this makes the bookkeeping easier, wish I had thought of it when I did my optimization pass.